docs/visual-basic/programming-guide/language-features/procedures/how-to-call-an-event-handler.md

```diff
@@ -1,25 +1,26 @@
 ---
-title: "How to: Call an Event Handler"
+title: "How to call an event handler"
```

Comment:
@tdykstra Your PR for ms.topic: how-to included several folders but not Visual Basic folder.
Do you want me to apply ms.topic globally or set it here explicitly